### DISCUSSION TITLE
command to enable premium account

### DISCUSSION THREAD

**Discussion by GPConnect** [simple-login/app#620] [Self-hosting question]
Hello, thanks for this useful software ! 
can i have an "copy/paste" command to enable premium on my account"  like the docs says ? 
i not too familiar with postgresql
"By default, new accounts are not premium so don't have unlimited alias. To make your account premium, please go to the database, table "users" and set "lifetime" column to "1" or "TRUE"."

Thanks

**Comment by nguyenkims** [comment]
You can log into your database by running `docker exec -it sl-db psql -U myuser simplelogin` and then run this query that would upgrade all users to the lifetime license:

```update users set lifetime=true;```
